README.md

```diff
@@ -1,30 +1,54 @@
-# source-verify
+# Sourcify 🧑‍💻📝🔍 
 
-Source-Verify is a service that verifies Ethereum bytecode was compiled from a certain
+Sourcify wants to help make contract interactions on the blockchain safer and more transparent for users.
+
+To achieve this goal, Sourcify supports several efforts to foster adoption of open-source source file verification, metadata files and NatSpec comments.
+
+At its core, Sourcify currently maintains
+
++ an [interface](https://github.com/sourcifyeth/sourcify/tree/master/ui) that helps developers to verify metadata and contract source code. It is accessible via [sourcify.dev](https://sourcify.dev/).
++ a decentralized contract repository of all verified contracts, powered by IPFS, accessible via [sourcify.dev](https://sourcify.dev/) and [verificat.eth](https://gateway.ipfs.io/ipfs/QmcgzQhyAXsLJ1nhZ4pPVVMqvoivpXynwdhHMx4J1xavqv/)(soon to be transfered to sourcify.eth).
```

Comment:
I think the link verificat.eth should point to verificat.eth or verificat.eth.link

Reply:
yes!